managed-services/iot-hub/quickstart.mdx

```diff
@@ -26,45 +26,45 @@ In this Quickstart, we show you how to create your first IoT Hub to connect obje
 ## How to create an IoT Hub
 
 1. Click **IoT Hub** in the **Managed Services** section of the side menu. If you have not already created a Hub, the IoT Hub creation page displays.
-2. Click **Create a Hub**. The hub creation wizard displays.
-3. Enter a name for your Hub and choose the geographical region to deploy your hub.
-4. Choose a product plan for your IoT Hub. Currently three product plans are available for Hubs:
+2. Click **Create hub**. The hub creation wizard displays.
+3. Enter a name for your hub and choose the geographical region to deploy your hub.
+4. Choose a product plan for your IoT Hub. Currently, three product plans are available for hubs:
     - **Shared plan:** This is a cost-effective, straightforward plan. Good for proofs-of-concept or DIY projects. The price is kept low by sharing resources between users, but it is therefore forbidden to use MQTT features which require memory (no messages with `QoS1`, `QoS2`, `retained` and no anonymous devices). Although this plan does not limit the number of devices, you will be billed for devices exceeding the number allowed within the free tier.
-    - **Dedicated Plan:** This plan offers dedicated resources for the Hub, which means more predictable performances and full support of the MQTT protocol.
-    - **High Availability (HA) Plan:** Not only are resources dedicated to the Hub, they are also replicated so that the broker is highly available. This setting is ideal for workloads that require greater robustness for their Hub.
-5. Click **Create a Hub** or **Create a Hub and Add a Device**.
+    - **Dedicated Plan:** This plan offers dedicated resources for the hub, which means more predictable performances and full support of the MQTT protocol.
+    - **High Availability (HA) Plan:** Not only are resources dedicated to the hub, they are also replicated so that the broker is highly available. This setting is ideal for workloads that require greater robustness for their Hub.
```

Comment:
```suggestion
    - **High Availability (HA) Plan:** Not only are resources dedicated to the hub, they are also replicated so that the broker is highly available. This setting is ideal for workloads that require greater robustness for their hub.
```